dev-libs/libpcre/libpcre-8.45.recipe

```diff
@@ -9,10 +9,11 @@ COPYRIGHT="1997-2018 University of Cambridge
 	2007-2012, Google Inc. All rights reserved."
 HOMEPAGE="http://www.pcre.org/"
 LICENSE="PCRE"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="https://ftp.pcre.org/pub/pcre/pcre-$portVersion.tar.bz2"
```

Comment:
SOURCE_URI has changed to: https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.bz2

Reply:
Shorter link: https://downloads.sourceforge.net/pcre//pcre-8.45.tar.bz2